recipe/meta.yaml

```diff
@@ -715,30 +892,13 @@ outputs:
         # https://conda-forge.org/docs/maintainer/knowledge_base.html#nvcuda-dll-cannot-be-found-on-windows
         # However, we check below for (at least) the presence of a correctly-compiled module
         - pyarrow.cuda     # [cuda_compiler_version != "None" and not win]
```

Comment:
Same here, no tests that we've already done, unless they capture a meaningful change between the various different layers.